src/components/components/Component.js

```diff
@@ -176,17 +229,66 @@ export default class Component extends React.Component {
         if (whichAssetType == "background"){
           whichOptions = this.state.backgroundList;
         }
+        const objId = this.props.entity.getAttribute("id");//.replace("-obj", "");
+        let isObjChecked = false;
+        if (objId in this.state.idToCheckedMap){
+          isObjChecked = this.state.idToCheckedMap[objId];
+        }
         return (
-          <Select
-            styles={customStyles}
-            value={this.state.nameList.filter(option => option.value == componentData.data)}
-            ref="select"
-            options={whichOptions}
-            placeholder="Add component..."
-            noResultsText="No components found"
-            searchable={true}
-            onChange={this.selectOption}
-          />
+          <div>
```

Comment:
Can you put this into another component like we talked about @amoliksingh? 
Like create a new React component, and put this inside its render method. For now, it is okay if it just hardcodes this exact HTML in there.
And then in here, instead of having this div HTML, you can just do <ComponentName />